vorbis_test.go

```diff
@@ -65,6 +65,8 @@ func TestVorbisFromExistingFlac(t *testing.T) {
 		}
 	}
 
+	cmt.Add(FIELD_GENRE, "Bee Pop")
```

Comment:
Please check for errors here.

Reply:
you are right. Could have thought about it 🤦‍♂ PR is updated